app/src/main/java/org/wikipedia/feed/model/UtcDate.kt

```diff
@@ -0,0 +1,42 @@
+package org.wikipedia.feed.model
+
+import java.util.Calendar
+import java.util.TimeZone
+
+class UtcDate(age: Int) {
+    private val cal: Calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"))
+    private val year: String
+    private val month: String
+    private val date: String
+
+    fun baseCalendar(): Calendar {
+        return cal
+    }
+
+    fun year(): String {
```

Comment:
None of these getter methods are necessary. That's one of the benefits of converting to Kotlin.

Reply:
Updated via 4ae3b1e